### ISSUE TITLE
I made a key map of the item dict:

### ISSUE THREAD

**Issue by Phillyclause89** [TheMannInBlack/Spotipy#1]
https://github.com/Phillyclause89/MySpotipy/wiki/Spotipy-Track-Data-Structure

Thought I'd let you know incase you wanted to get more than just track and artist names into your csv.

**Comment by TheMannInBlack** [comment]
Thanks, that’s really helpful. I saw this on the Spotify API website, but Python is the only language I’m familiar with. Attempting to decipher it and then use it was challenging, but this is much clearer.

**Comment by Phillyclause89** [comment]
Yeah, found out during testing that  I messed up of some of the keys near the bottom of the page. they are missing ['track'] level.
